snap/hooks/configure

```diff
@@ -62,6 +62,12 @@ vt=0
 EOT
 fi
 
+
+if [ -e "$SNAP/graphics/lib/mir-quirks" ]
```

Comment:
We might want this to use the `-f` test? We don't want to accidentally pick up a `mir-quirks` directory.